DPGAnalysis/SiStripTools/plugins/APVCyclePhaseProducerFromL1ABC.cc

```diff
@@ -192,24 +198,43 @@ void APVCyclePhaseProducerFromL1ABC::produce(edm::Event& iEvent, const edm::Even
     long long orbitoffset = _orbitoffsetSOR;
     int bxoffset = 0;
 
-    for (L1AcceptBunchCrossingCollection::const_iterator l1abc = pIn->begin(); l1abc != pIn->end(); ++l1abc) {
-      if (l1abc->l1AcceptOffset() == 0) {
-        if (l1abc->eventType() != 0) {
-          orbitoffset = (long long)iEvent.orbitNumber() - (long long)l1abc->orbitNumber();
-          bxoffset = iEvent.bunchCrossing() - l1abc->bunchCrossing();
-
-          if (_wantHistos) {
-            if (_hbx && *_hbx)
-              (*_hbx)->Fill(l1abc->bunchCrossing());
-            if (_hdbx && *_hdbx)
-              (*_hdbx)->Fill(bxoffset);
-            if (_hdorbit && *_hdorbit)
-              (*_hdorbit)->Fill(orbitoffset);
-          }
-        } else {
-          edm::LogWarning("L1AcceptBunchCrossingNoType") << "L1AcceptBunchCrossing with no type found: ";
-          for (L1AcceptBunchCrossingCollection::const_iterator debu = pIn->begin(); debu != pIn->end(); ++debu) {
-            edm::LogPrint("L1AcceptBunchCrossingNoType") << *debu;
+    try {
```

Comment:
I think this goes against the code guidelines: http://cms-sw.github.io/cms_coding_rules.html (section 7, point 11)

Reply:
Fixed (will push in a while)